features/entering-rooms.feature

```diff
@@ -16,19 +16,19 @@ Feature: Entering Rooms
     When the Workspace Member taps the "Listed Room 1" Room in the Room Picker
     Then the Workspace Member is placed in the "Listed Room 1" Room
 
-  @built @unimplemented-steps
+  @built
   Scenario: Entering Room via Room Picker from another Room
-    Given I am in a Room
-    When I tap a different Room in the Room Picker
-    Then I am in the Room
+    Given the Workspace Member is in the "System Test" Workspace and in the "Listed Room 1" Room
```

Comment:
Good call on moving this to the Workspace Member instead of I to make it more clear who the actor is.